gui/src/avplacebowidget.cpp

```diff
@@ -44,6 +44,9 @@ AVPlaceboWidget::AVPlaceboWidget(
         CHIAKI_LOGI(session->GetChiakiLog(), "Using placebo high quality preset");
         render_params = pl_render_high_quality_params;
     }
+
+    QImageReader logoReader(":icons/chiaki.svg");
```

Comment:
It's SVG image so that's why QImageReader was used to always render the SVG at requested size. So now it will be blurry due to scaling in RenderPlaceholderIcon.

Reply:
I tested it on my display and it didn't seem that blurry to me, but I'll revert it. It's not like we're redrawing that logo every frame anyway.